tests/unit/logs/test_logs_sender.py

```diff
@@ -72,6 +72,8 @@ def test_request_and_content_length(self):
 
         logs_sender.DYNATRACE_LOG_INGEST_CONTENT_MAX_LENGTH = 50
         logs_sender.DYNATRACE_LOG_INGEST_REQUEST_MAX_SIZE = 115
+        logs_sender.DYNATRACE_LOG_INGEST_CONTENT_MARK_TRIMMED = "[TRUNCATED]"
```

Comment:
Is this necessary? The value is the same as the one in `logs_sender.py`.

Reply:
I don't know if I get the point, but in my opinion, it is absolutely OK for the test to break when changes in source code happen.

Reply:
I agree with @bmrozinski-dt 
I didn't notice this at first. We definitely want to be informed through tests what functionality will change/break if we change something in the production code. So if we change it in `logs_sender.py` to `[TRUNCATED BY DYNATRACE]`, we want this test to fail and we'll adjust it accordingly